File: specs/master/document/multiple/multi_term_vectors/multi_term_vectors_request.ts

/**namespace:Document.Multiple.MultiTermVectors */
interface MultiTermVectorsRequest extends Request {
	docs: MultiTermVectorOperation[];
	/**ambiguous_origin*/
	TermStatistics: boolean;
	/**ambiguous_origin*/
	FieldStatistics: boolean;
	/**ambiguous_origin*/
	Fields: Field[];
	/**ambiguous_origin*/
	Offsets: boolean;
	/**ambiguous_origin*/
	Positions: boolean;
	/**ambiguous_origin*/
	Payloads: boolean;
	/**ambiguous_origin*/
	Preference: string;
	/**ambiguous_origin*/
	Routing: string;
	/**ambiguous_origin*/
	Parent: string;
	/**ambiguous_origin*/
	Realtime: boolean;
	/**ambiguous_origin*/
	Version: long;
	/**ambiguous_origin*/
	VersionType: VersionType;
	/**ambiguous_origin*/
	Source: string;
	/**ambiguous_origin*/
	FilterPath: string;
}

File: output/csharp/Document/Multiple/MultiTermVectors/MultiTermVectorsRequest.cs
using Nest.CommonAbstractions;
using Nest.Internal;
using Nest.Common;
using Nest.Document;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Document {

	public class MultiTermVectorsRequest : RequestBase {
		
		[DataMember(Name="fields")]
		public List<Field> Fields { get; set; }


		[DataMember(Name="field_statistics")]
		public bool FieldStatistics { get; set; }


		[DataMember(Name="offsets")]
		public bool Offsets { get; set; }


		[DataMember(Name="payloads")]
		public bool Payloads { get; set; }


		[DataMember(Name="positions")]
		public bool Positions { get; set; }


		[DataMember(Name="preference")]
		public string Preference { get; set; }


		[DataMember(Name="realtime")]
		public bool Realtime { get; set; }


		[DataMember(Name="routing")]
		public Routing Routing { get; set; }


		[DataMember(Name="term_statistics")]
		public bool TermStatistics { get; set; }


		[DataMember(Name="version")]
		public long Version { get; set; }


		[DataMember(Name="version_type")]
		public VersionType VersionType { get; set; }


		[DataMember(Name="docs")]
		public List<MultiTermVectorOperation> Docs { get; set; }


		[DataMember(Name="ids")]
		public List<Id> Ids { get; set; }

	}
}
